src/vs/workbench/browser/parts/editor/media/tabstitlecontrol.css

```diff
@@ -108,20 +113,12 @@
 	display: none;
 }
 
-.monaco-workbench .part.editor > .content .editor-group-container > .title .tabs-container > .tab:first-of-type {
-	padding-left: 5px !important;
-}
-
-.monaco-workbench .part.editor > .content .editor-group-container > .title .tabs-container > .tab:last-of-type {
-	padding-right: 5px !important;
+.monaco-workbench .part.editor > .content .editor-group-container.active > .title .tabs-container > .tab:hover > .tab-actions .action-label {
+	border-radius: 50%;
 }
 
 .monaco-workbench .part.editor > .content .editor-group-container > .title > .tabs-and-actions-container.wrapping .tabs-container > .tab:last-child {
-	margin-right: var(--last-tab-margin-right); /* when tabs wrap, we need a margin away from the absolute positioned editor actions */
-}
-
-.monaco-workbench .part.editor > .content .editor-group-container > .title > .tabs-and-actions-container.wrapping .tabs-container > .tab.last-in-row:not(:last-child) {
```

Comment:
Custom CSS does not substitute the CSS files themselves. If we remove the selector in this scenario, it will not be removed from vscode. Our objective should be to override its properties in order to disable it. For instance, we can use display: none or an equivalent method.